docs/metrics.md

```diff
@@ -116,8 +116,10 @@ scp.pending.ready                        | counter   | number of envelopes ready
 scp.sync.lost                            | meter     | validator lost sync
 scp.timeout.nominate                     | meter     | timeouts in nomination
 scp.timeout.prepare                      | meter     | timeouts in ballot protocol
-scp.timing.externalized                  | timer     | time spent in ballot protocol
 scp.timing.nominated                     | timer     | time spent in nomination
+scp.timing.externalized                  | timer     | time spent in ballot protocol
+scp.timing.externalize-lag               | timer     | delay between first externalize message received and node externalizing
+scp.timing.externalize-delay             | timer     | delay between node externalizing and other externalize messages
```

Comment:
Can we slightly tweak this description to make it clear that "other externalize messages" are the externalize messages received after the node itself externalizes?

Reply:
done

Reply:
Looks like the description is still the same?

Reply:
I thought you were asking about "node externalizing" that I clarified as "local node externalizing", but yeah I replaced "other" with "later"